src/homepage/Components/Banner.js

```diff
@@ -33,25 +93,29 @@ export default function Banner() {
           </Col>
         </Row>
       </Container>
+
       <Container className={styles.containersmall} fluid>
         <h1 style={{ textAlign: "center" }} className={styles.heading}>
-          Give wings to <br className={styles.align} /> your
-          <span style={{ color: "#008dc8" }}> dreams </span>
+          From Will to <br className={styles.align} /> to
```

Comment:
From Will to **to** Skill.

`To` is two times here

Reply:
Have a look now.

Reply:
The layout "From Will to Skill" needs to be revised, because the text is small, it isn't looking good on the banner. 

Let's do one thing, Revert all your changes in the banner, (Keep it the original one - Give wings to your dreams), and just give social media icons in this PR. I will merge right away.